Translations/fr.po

```diff
@@ -6448,81 +6245,65 @@ msgstr ""
 
 #. TRANSLATORS: Quest Map
 #: Source/quests.cpp:92
-#, fuzzy
 msgid "A Dark Passage"
-msgstr "l'obscurité"
+msgstr "Un Passage Sombre"
 
 #. TRANSLATORS: Quest Map
 #: Source/quests.cpp:93
-#, fuzzy
 msgid "Unholy Altar"
 msgstr "L'Autel impie"
 
 #. TRANSLATORS: Used for Quest Portals. {:s} is a Map Name
 #: Source/quests.cpp:449
-#, fuzzy
 msgid "To {:s}"
-msgstr "{:s} Compétence"
+msgstr "De {:s}"
 
 #: Source/setmaps.cpp:24
-#, fuzzy
 msgid "Skeleton King's Lair"
-msgstr "Du Roi"
+msgstr "L'Antre du Roi Squelette"
 
 #: Source/setmaps.cpp:25
-#, fuzzy
 msgid "Chamber of Bone"
-msgstr "Armure en Mailles d'os"
+msgstr "Chambre d'os"
 
 #: Source/setmaps.cpp:28
-#, fuzzy
 msgid "Archbishop Lazarus' Lair"
-msgstr "Bâton de Lazare"
+msgstr "L'Antre de l’Archevêque Lazarus"
 
 #: Source/spelldat.cpp:16
-#, fuzzy
-#| msgid "Fire "
 msgctxt "spell"
 msgid "Firebolt"
-msgstr "Feu "
+msgstr "[sort] Feu "
```

Comment:
Why is all of this prefixed with [sort]?

Reply:
Because I get fooled by my tool :D